src/dialog.tsx

```diff
@@ -162,9 +162,9 @@ const Dialog: React.FC<DialogProps> = (props) => {
             {icon && <Box paddingBottom={24}>{icon}</Box>}
             {title && (
                 <Box paddingBottom={16}>
-                    <Text4 as="h2" light>
+                    <Text5 as="h2" light>
```

Comment:
remove the light prop, Text5 can only be light
```suggestion
                    <Text5 as="h2">
```